### PR TITLE
Fix issue with borders in Devices and other views

### PR DESCRIPTION
**Pull request by ct-goranpalibrk**
Fixes issue #1323

### CODE REVIEW

**Comment by LeoNerdoG** [comment]
I have verified that this PR resolves issue #1323.

File: console/module/account/src/main/java/org/eclipse/kapua/app/console/module/account/client/AccountDetailsTabDescription.java [hunk 1]
```diff
@@ -105,4 +106,11 @@ public void onSuccess(GwtAccount result) {
             }
         });
     }
+
+    @Override
+    protected void onRender(Element parent, int index) {
+        // TODO Auto-generated method stub
```
Comment (muros-ct) [inline console/module/account/src/main/java/org/eclipse/kapua/app/console/module/account/client/AccountDetailsTabDescription.java]: Please remove generated TODO or add missing code.

File: console/module/authorization/src/main/java/org/eclipse/kapua/app/console/module/authorization/client/tabs/role/UserTabAccessRoleGrid.java [hunk 2]
```diff
@@ -118,21 +118,15 @@ public void setFilterQuery(GwtQuery filterQuery) {
...
             toolbar.setEditButtonVisible(false);
-            toolbar.setBorders(false);
+            toolbar.setBorders(true);
         }
         return toolbar;
     }
 
     @Override
     protected void onRender(Element target, int index) {
+        // TODO Auto-generated method stub
```
Comment (muros-ct) [inline console/module/authorization/src/main/java/org/eclipse/kapua/app/console/module/authorization/client/tabs/role/UserTabAccessRoleGrid.java]: Remove TODO or add missing code.

**Comment by ct-goranpalibrk** [comment]
To Do in onRender method removed from UserTabAccessRoleGrid and from AccountDetailsTabDescription.

**Pull request merged**